packages/cli/tasks-info.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  TASK_INTERACT_NAME: 'interact',
+  TASK_INTERACT_DESC: 'Spins up an interactive ui to interact with contracts.',
+
+  TASK_NOMINATE_OWNER_NAME: 'nominate-owner',
```

Comment:
@leomassazza what do you think about removing all unimplemented tasks for now? Then we can focus on the interactive task fully and interpret anything present as functional.

Reply:
ok